driver-core/src/test/unit/com/mongodb/ConnectionStringSpecification.groovy

```diff
@@ -601,7 +601,7 @@ class ConnectionStringSpecification extends Specification {
         new ConnectionString('mongodb://jeff@localhost/?' +
                              'authMechanism=GSSAPI' +
                              '&authMechanismProperties=' +
-                             'SERVICE_NAME:foo:bar')
+                             'SERVICE_NAMEbar') // missing =
```

Comment:
No spec was associated with the Jira ticket where this check was added. OIDC tests include a value with a second `:` (when decoded). Rather than remove, I replaced with a different invalidity check.